exdb/static/exdb/css/completion_board.css

```diff
@@ -0,0 +1,35 @@
+td, th{
+    border: 1px solid black;
+    text-align: center;
+}
+
+.cell {
+    visibility: hidden;
+    background-color: #fff;
+    box-shadow: 0 0 20px 12px #fff;
+}
+
+td:hover .cell {
+    visibility: visible;
+}
+
+.requirement-header{
```

Comment:
It looks like `requirement-header` and `requirement-head` can be merged into one.